README.adoc

```diff
@@ -4404,18 +4404,25 @@ batman.fetch(:powers, obtain_batman_powers) # obtain_batman_powers is an expensi
 batman.fetch(:powers) { obtain_batman_powers }
 ----
 
-=== `Hash#values_at` [[hash-values-at]]
+=== `Hash#values_at` [[hash-values-at-and-fetch-values]]
```

Comment:
You should check if something in RuboCop doesn't refer to this anchor before changing it.

Reply:
Sure, there are no links to this anchor.